Cargo.toml

```diff
@@ -14,16 +15,16 @@ repository = "https://github.com/keep-starknet-strange/cairo-lint"
 license-file = "LICENSE"
 
 [workspace.dependencies]
-cairo-lang-compiler = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-semantic = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-filesystem = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-diagnostics = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-test-plugin = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-syntax = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-defs = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
-cairo-lang-test-utils = { git = "https://github.com/starkware-libs/cairo", rev = "ca2dceb" }
+cairo-lang-compiler = "*"
```

Comment:
we will also need to synchronize cairo-lint version with compiler's one

Reply:
Set the cairo-lint's version to 2.9.2.

Reply:
I am not sure what do you exactly mean by that? Setting the `cairo-toolchain-xtasks` to 1.0.1 and our xtask package version to 1.0.1?

Reply:
no, I meant this: https://github.com/software-mansion/scarb/blob/main/xtask/src/sync_version.rs

Reply:
Added the xtask 👍